webview/cocoa.py

```diff
@@ -253,7 +262,7 @@ def show(self):
             BrowserView.app.run()
 
     def destroy(self):
-        BrowserView.app.stop_(self)
+        PyObjCTools.AppHelper.callAfter(self.window.close)
```

Comment:
You mean the window created first? It will just close. The app terminates only
when all windows of the app are closed.

This is in accordance with cuibonobo's [comment](https://github.com/r0x0r/pywebview/issues/43#issue-171017765), who suggested the idea to
begin with. The code that quits the app is in `windowWillClose` method of the
`WindowDelegate` ([here](https://github.com/r0x0r/pywebview/commit/d771625b42ac8917b6f3bdb61a1a686803809643#diff-2ec81f56c33197a32ca5ae4c092a6a2aL53)).

The current design makes no distinction between 'parent' or 'child' windows.
All windows, whether created by Python or JS, are 'children' of the app. You
want `pywebview` to give the initial window some special status? So that the
app terminates when it is closed? If this is the desired behaviour, it can be
implemented easily, I think. There is a `mainWindow` property to the app that
we can utilise. But the official documentation for the JS `window.open` makes
no such requirement. Try `window.open` in a browser, and the child window
stays open even after closing the browser's main window.